packages/client/merge/README.md

```diff
@@ -0,0 +1,93 @@
+2. Or the CL starts issuing engine API calls transitioning the normal forward sync into (backfill) beacon sync, signalling to ethereumjs that the chain is post merge. 
+
+So to run ethereumjs in PoS (ready) configurations, one requires to specify the correct ttd in the network configurations as well as turn on the engine api (via `--rpcEngine`). 
+
+## Networks which have been merged to PoS
+
+Following networks have been merge to PoS with ethereumjs client having the baked in `merge` hardfork configurations:
+1. Ropsten (`--network ropsten`)
+2. Sepolia (`--network sepolia`)
+
+The `goerli` and `mainnet` hardforks will be configured for merge in the future releases. We recommend running `ethereumjs` on `sepolia` network as its relatively young and light (and the `kiln` network will be deprecated soon).
+
+## Sepolia testnet instructions
+
+- Execution chain can be followed at: https://sepolia.etherscan.io
+- Beacon chain can be followed at: https://sepolia.beaconcha.in
+
+To run `sepolia`, the configurations are baked in for both EL and CL clients. Our instructions will focus on the `lodestar` CL client, however other CL clients (`lighthouse`,`nimbus`,`teku`,`prysm`) can be similarly hooked up to `ethereumjs` as they all (should) follow `engine` API spec. 
+
+### Jwt-secret
+The EL (`ethereumjs`) and CL (`lodestar`) communications over the `engine` API are protected by `jwt` auth. To enable this, you will need to generate and provide a jwt secret. We just use here a dummy one for illustrations, please generate one of your own. 
+
+```shell
+echo "0xdc6457099f127cf0bac78de8b297df04951281909db4f58b43def7c7151e765d" > /path/to/jwtsecretfile
+```
+
+### Execution - EthereumJS Setup
+
+Please ensure you have Node 16.x installed.
+
+#### Client Installation
+
+##### GitHub
+
+1. `git clone --depth 1 --branch master https://github.com/ethereumjs/ethereumjs-monorepo.git`
+2. `cd ethereumjs-monorepo`
+3. `npm i`
+4. `cd packages/client`
+
+##### NPM
+
+```shell
+npm install -g @ethereumjs/client
+```
+
+Note that you eventually want to start the client just with `ethereumjs` instead of using the `npm run client:start` GitHub start command (also leave the in-between `--` to forward the client config options).
+
+##### Docker
+
+Or try it in Docker.
+
+In `packages/merge`, you can build a docker image using
+```shell
+docker build . --tag ethereumjs:latest
+```
+You may now do appropriate directory/file mounts for `data` dir and `jwtsecret` file and provide their path appropriately in the `client` run command.
+
+#### Run client
+
+This the the bare minimum command required for interop with the CL `lodestar`. You may add further args to expose functionality for your use case.
+
+```shell
+npm run client:start -- --network sepolia --datadir /path/to/sepolia/data/dir --rpcEngine --jwt-secret /path/to/jwtsecret/file
```

Comment:
I will just ask some questions here, otherwise merge.

Depending on the answers I would then directly integrate/update along the release PR.

So here: do we maybe want to skip the `--datadir` option now? Same for Lodestar? This is now totally optional and if people have any datadir change needs then they are just in the mode of "unrelatedly using the datadir option".

Reply:
Second question on this: is the JWT secret generated on first usage when used like this?

If not: then I would prefer to describe a slightly alternative way to use (so: EthereumJS without the option -> Lodestar taking over the path). Guess otherwise people might be a bit lost here.

If yes: then I would like to add a sentence on this.

Reply:
>So here: do we maybe want to skip the --datadir option now?
yes we may skip :+1:

Reply:
>Second question on this: is the JWT secret generated on first usage when used like this? 
I added a jwt-secret step (in above section) although we generate one randomly and use it, which can then be provided to lodestar, but from what i have seen, many people get confused here and its better to just have an explicit instruction.